example/src/demos/htmlCluster.tsx

```diff
@@ -3,7 +3,10 @@ import ReactMapboxGl, { Marker, Cluster, Popup } from '../../../';
 import styled from 'styled-components';
 
 // tslint:disable-next-line:no-var-requires
-const { token, styles: { outdoor } } = require('./config.json');
+const {
+  token,
+  styles: { outdoor }
+} = require('./config.json');
```

Comment:
unrelated to the PR really, but adding a note: should use TypeScript's `resolveJsonModule` option for doing these imports